build/stf-run-ci/tasks/setup_stf_local_build.yml

```diff
@@ -53,14 +57,26 @@
       OPERATOR_IMAGE={{ sto_image_path | parse_image | quote }} \
       OPERATOR_TAG={{ sto_image_path | parse_tag | quote }} \
       ./generate_bundle.sh
+  register: generate_bundle_sto
+
+- name: Results of STO bundle generation
+  debug:
+    var: generate_bundle_sto.stdout
```

Comment:
Above you used `generate_bundle_sto` (without the .stdout)

Reply:
Thanks for the catch! I will make this consistent.

Reply:
Looking at the contents here, I'm going to switch to using `.stdout` in both places for debugging.

```bash
TASK [stf-run-ci : Results of SGO bundle generation] *********************************************************************************************************************************************************************************************
ok: [localhost] => {
    "generate_bundle_sgo": {
        "changed": true,
        "cmd": "WORKING_DIR=\"/home/leif/src/github.com/infrawatch/service-telemetry-operator/build/working/smart-gateway-operator-bundle\" \\\nRELATED_IMAGE_CORE_SMARTGATEWAY=image-registry.openshift-image-registry.svc:5000/service-telemetry/sg-core \\\nRELATED_IMAGE_BRIDGE_SMARTGATEWAY=image-registry.openshift-image-registry.svc:5000/service-telemetry/sg-bridge \\\nRELATED_IMAGE_CORE_SMARTGATEWAY_TAG=latest \\\nRELATED_IMAGE_BRIDGE_SMARTGATEWAY_TAG=latest \\\nOPERATOR_IMAGE=image-registry.openshift-image-registry.svc:5000/service-telemetry/smart-gateway-operator \\\nOPERATOR_TAG=latest \\\n./generate_bundle.sh\n",
        "delta": "0:00:00.384883",
        "end": "2023-06-06 15:14:31.197194",
        "failed": false,
        "msg": "",
        "rc": 0,
        "start": "2023-06-06 15:14:30.812311",
        "stderr": "",
        "stderr_lines": [],
        "stdout": "{\"operator_bundle_image\":\"quay.io/infrawatch-operators/smart-gateway-operator-bundle\",\"operator_bundle_version\":\"5.0.1686078870\",\"operator_image\":\"image-registry.openshift-image-registry.svc:5000/service-telemetry/smart-gateway-operator\",\"bundle_channels\":\"unstable\",\"bundle_default_channel\":\"unstable\",\"operator_tag\":\"latest\",\"working_dir\":\"/home/leif/src/github.com/infrawatch/service-telemetry-operator/build/working/smart-gateway-operator-bundle\"}",
        "stdout_lines": [
            "{\"operator_bundle_image\":\"quay.io/infrawatch-operators/smart-gateway-operator-bundle\",\"operator_bundle_version\":\"5.0.1686078870\",\"operator_image\":\"image-registry.openshift-image-registry.svc:5000/service-telemetry/smart-gateway-operator\",\"bundle_channels\":\"unstable\",\"bundle_default_channel\":\"unstable\",\"operator_tag\":\"latest\",\"working_dir\":\"/home/leif/src/github.com/infrawatch/service-telemetry-operator/build/working/smart-gateway-operator-bundle\"}"
        ]
    }
}
```

vs

```bash
TASK [stf-run-ci : Results of STO bundle generation] *********************************************************************************************************************************************************************************************
ok: [localhost] => {
    "generate_bundle_sto.stdout": {
        "bundle_channels": "unstable",
        "bundle_default_channel": "unstable",
        "operator_bundle_image": "quay.io/infrawatch-operators/service-telemetry-operator-bundle",
        "operator_bundle_version": "1.5.1686078872",
        "operator_image": "image-registry.openshift-image-registry.svc:5000/service-telemetry/service-telemetry-operator",
        "operator_tag": "latest",
        "working_dir": "/home/leif/src/github.com/infrawatch/service-telemetry-operator/build/working/service-telemetry-operator-bundle"
    }
}
```